enhancements/network/netflow.md

```diff
@@ -0,0 +1,107 @@
+---
+title: OpenShift Network Flow export and collection support
+authors:
+  - "@rcarrillocruz"
+reviewers:
+  - "@russellb"
+approvers:
+  - TBD
+creation-date: 2021-01-11
+last-updated: 2020-01-11
+status: provisional
+---
+
+# OpenShift NetFlow support
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+OVS (Open vSwitch) supports exporting network flow protocols like NetFlow, IPFIX and sflow.
+OpenShift uses OVN/OVS for OVN-Kubernetes network plugin. 
+OpenShift should have the ability to export cluster traffic using any of this protocols.
+
+## Motivation
+
+* Customers need to be able to get network flows out of OpenShift to be consumed by their collectors.
+* Customers need to be able to see a log of the network traffic that flows thru OpenShift cluster.
+* Customers need to get metrics and monitoring data of the OpenShift cluster network traffic to make decisions on capacity and/or security issues.
+
+### Goals
+
+- Add support for exporting all the network flows protocols supported by OVS from OpenShift.
+- Add an optional collector that consumes the network flows data and enriches it with Kubernetes metadata. 
+- Add Prometheus metrics from the data collected by the optional collector for visualization and alerting purposes.
+
+### Non-Goals
```

Comment:
Another non-goal to call out is that we're not doing any visualization in this first step

Reply:
Ack, will add.

Reply:
done